Add Chainstack 3.0 changes (#460)
chainstack/docs: docs/operations/ethereum/debug-and-trace-apis.md

---
meta:
  - name: description

>     content: Learn what the debug and tracing API is and how to trace transactions on Ethereum.

Reviewer: ```suggestion
    content: Learn what the debug and trace APIs are and how to trace transactions on Ethereum.
```
?

An elastic Ethereum archive node deployed as Erigon exposes the `debug_*` and `trace_*` API methods.

For the full list of the available debug and trace API methods, see [Erigon: RPC implementation status](https://github.com/ledgerwatch/erigon/blob/stable/cmd/rpcdaemon/README.md#rpc-implementation-status).

## Dedicated nodes

You can deploy [dedicated](/glossary/dedicated-node) Ethereum archive nodes starting from the <a href="https://chainstack.com/pricing/" target="_blank">Growth plan</a> as the Geth or Erigon [client implementation](/operations/ethereum/clients).

A dedicated Ethereum archive node deployed as Geth exposes the `debug_*` API methods.

A dedicated Ethereum archive node deployed as Erigon exposes the `debug_*` and `trace_*` API methods.

For the full list of the available debug and trace API methods, see:

* [Geth: debug namespace](https://geth.ethereum.org/docs/rpc/ns-debug)
* [Erigon: RPC implementation status](https://github.com/ledgerwatch/erigon/blob/stable/cmd/rpcdaemon/README.md#rpc-implementation-status)

## Usage example

> You can debug and trace transactions by replaying them in the Ethereum Virtual Machine to get the execution details in the exact same way as they happened on the chain.

Reviewer: Example for `debug_traceBlockByNumber` could be useful as it's quite popular call

Author: Changed.
